src/main/java/org/fcrepo/storage/ocfl/DefaultOcflObjectSession.java

```diff
@@ -484,6 +507,39 @@ private void writeHeaders(final ResourceHeaders headers, final Path destination)
         }
     }
 
+    private void touchRelatedResources(final ResourceHeaders headers) {
+        // Touch the AG for non-ACL AG part updates
+        if (isArchivalGroup
+                && !Objects.equals(rootResourceId(), headers.getId())
+                && !InteractionModel.ACL.getUri().equals(headers.getInteractionModel())) {
```

Comment:
This is the way that Fedora currently works. The reason is that ACLs are never versioned from Fedora's perspective. That is to say, when accessing a memento, the most recent ACL is always applied.

Reply:
Hmm, okay, it will still be versioned but that's invisible from fedora perspective I guess, so its probably reasonable for it to not be reflected on the versions of the AG. We don't need to be able to surface the historic version of acls as they existed at a particular version of the AG, so unless versioning of ACLs from the fedora API perspective becomes a use case, then this should be fine.